src/mscorlib/shared/System/StringComparer.cs

```diff
@@ -228,7 +229,85 @@ public override int GetHashCode()
 
     [Serializable]
     [System.Runtime.CompilerServices.TypeForwardedFrom("mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
-    internal sealed class OrdinalComparer : StringComparer 
+    internal class OrdinalComparer : StringComparer 
+    {
+        private bool _ignoreCase;
```

Comment:
It doesn't have one in netfx (this is the port from it). The type doesn't implement ISerializable so why would we need one?